src/libraries/System.Text.Json/tests/Serialization/ReadValueTests.cs

```diff
@@ -33,7 +33,7 @@ public static void NullReturnTypeThrows()
         [Fact]
         public static void NullJsonThrows()
         {
-            ArgumentNullException ex = Assert.Throws<ArgumentNullException>(() => JsonSerializer.Deserialize(json: null, returnType: typeof(string)));
```

Comment:
Please retain this test (in addition to the new one) and assert on the new expected behavior.

Reply:
I'm not removing this test. Not adding the conversion will cause ambiguous because `null` can be implicitly converted to span too.